include/pm/device_runtime.h

```diff
@@ -61,9 +61,10 @@ int pm_device_runtime_disable(const struct device *dev);
  *
  * @param dev Device instance.
  *
- * @retval 0 If the device has been resumed successfully.
- * @retval -ENOSTUP If runtime PM is not enabled for the device.
- * @retval -ENOSYS If the functionality is not available.
+ * @retval >0 This function will return the current device usage count (> 0) in
+ * case it succeeds. The return value 1 indicates that the device has just been
+ * resumed (first user). In case device runtime PM is not enabled, this function
+ * will always return 1.
```

Comment:
I don't think it should return "1" if device runtime pm is not enabled. 1 with this change means success for a get, so who calls it will assume that the device is active and that may not be the case ...

Reply:
`pm_device_runtime_disable` leaves devices active. When using runtime PM the device state should be controlled by the runtime API. If a user decides to change the device state manually, it’s his responsibility to control when a device driver can be called or not. The API has its usage guidelines, if you do not follow them you should really know what you're doing.

Reply:
that is not my point. I'm thinking in the path that the target was built without `CONFIG_PM_DEVICE_RUNTIME` and someone calls `pm_device_runtime_enable()` followed by `pm_device_runtime_get()`. Returning `1` may be interpreted that the device is active, but that is not true, the device may have not being initialized and not be active.